controlplane/eks/api/v1beta1/awsmanagedcontrolplane_types.go

```diff
@@ -224,6 +224,8 @@ type EncryptionConfig struct {
 type OIDCProviderStatus struct {
 	// ARN holds the ARN of the provider
 	ARN string `json:"arn,omitempty"`
+	// IssuerURL holds the OIDC Issuer URL of the cluster.
+	IssuerURL string `json:"issuerUrl,omitempty"`
```

Comment:
This doesn't need adding to v1beta1. Any new fields should just be on the latest apiversion.